design/templates/solution_design.md

```diff
@@ -0,0 +1,71 @@
+# Solution Design - Template
```

Comment:
does this indicate your branch may need a rebase?

Reply:
correct